recipe/migrations/openssl3.yaml

```
@@ -0,0 +1,7 @@
+__migrator:
+  build_number: 1
+  kind: version
+  migration_number: 1
+migrator_ts: 1631932209.13688
+openssl:
+- '3'
```

@conda-forge/core Any opinions on how to handle this one?

I'm not sure how easily it will go through but it might be smother if we build with both `1.1.1x` and `3.x`:

```suggestion
- 1.1.1
- '3'
```

I'm also not sure how to handle the removal of `pin_run_as_build` as OpenSSL 3.x now uses semantic versioning and `run_exports`.